Classes/common/CDTIncrementalStore.m

```diff
@@ -27,7 +27,7 @@
 #import "CDTISGraphviz.h"
 
 #ifdef HAS_NSBatchUpdateRequest
-/**
```

Comment:
I removed the double asterisk from internal methods to prevent them from showing up in the Cocoadocs documentation.  Cocoadocs, as I understand it, runs all the code through appledoc -- both the .h and .m files.  This means that internal methods decorated with the double asterisk show up in the external docs, which seems really unpleasant to me.  If we can find a way of excluding them from the Cocoadocs, I'll be happy to add the double asterisk back on all the internal methods.